Open Judge System/Workers/OJS.Workers.ExecutionStrategies/JavaSpringAndHibernateProjectExecutionStrategy.cs

```diff
@@ -279,7 +278,7 @@ protected void ExtractPackageAndDirectoryNames(string submissionFilePath)
 
             this.MainClassFileName = this.MainClassFileName.Substring(
                 this.MainClassFileName.LastIndexOf(".", StringComparison.Ordinal) + 1)
-                + GlobalConstants.JavaSourceFileExtension;
+                + Constants.JavaSourceFileExtension;
```

Comment:
Although I like this more, I believe the established convention, at least in **SULS** is to put operators on the above line:
``` 
this.MainClassFileName = this.MainClassFileName.Substring(
    this.MainClassFileName.LastIndexOf(".", StringComparison.Ordinal) + 1) + 
    Constants.JavaSourceFileExtension;